langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/internal/api/MistralAiChatCompletionRequest.java

```diff
@@ -33,4 +26,336 @@ public class MistralAiChatCompletionRequest {
     private List<MistralAiTool> tools;
     private MistralAiToolChoiceName toolChoice;
     private MistralAiResponseFormat responseFormat;
+
```

Comment:
I think that would be great if we can keep this following order for each API POJO, I'll take `MistralAiChatCompletionRequest` as example :

```java
@JsonDeserialize(builder = MistralAiChatCompletionRequest.Builder.class)
@JsonInclude(NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonNaming(SnakeCaseStrategy.class)
public class MistralAiChatCompletionRequest {
    private String model;
    private List<MistralAiChatMessage> messages;
    // rest of attributes here
    
    private  MistralAiChatCompletionRequest(Builder builder) {
          this.model = builder.model;
          this.messages = builder.messages;
          // rest of attributes here
    }
    
    public String model() {
           return model;
    }
    
    public List<MistralAiChatMessage> messages() {
           return messages;
    }
    // rest of attributes here
   
    @Override
    public boolean equals(Object another) {
          // implementation here
    }
   
    @Override
    public int hasCode() {
          // implementation here
    }
   
    @Override
    public String toString() {
          // implementation here
    }
    
    // builder pattern
    public static Builder builder() {
           return new Builder();
    }
  
    @JsonPOJOBuilder(withPrefix = "")
    @JsonIgnoreProperties(ignoreUnknown = true)
    @JsonNaming(SnakeCaseStrategy.class)
    public static final class Builder {
           private String model; // you can set a default value here.
           private List<MistralAiChatMessage> messages;
           // rest of attributes here
           
           private Builder() {}
           
           public Builder model(String model) {
                  this.model = model;
                  return this;
           }

           public Builder messages(List<MistralAiChatMessage> messages) {
                  this.messages = messages;
                  return this;
           }
           // rest of attributes here
           
          public MistralAiChatCompletionRequest build() {
                  return new MistralAiChatCompletionRequest(this);
          }
    }
}
```

would be great if you can apply it for each POJO.

Reply:
you mean to move the getters before the builder ?

Reply:
still in progress

Reply:
@czelabueno It should be done but I couldn't test it

Reply:
@ehsavoie the changes is not just moving the getters, please review it carefully, I see that you still do not accept the code suggestions and adapt the classes in the folder `model/mistralai`

Once you make these changes I could test it from your repo.

Reply:
Just to clarify, all classes under the folder `model/mistralai/internal/api` should follow the same order, not just the example

Reply:
@czelabueno I didn't accept them because I'm not familiar with this github feature so I made the changes manually